.gitignore

```diff
@@ -1,6 +1,6 @@
-# directories checked out by manage_externals, and other files created
-# by manage_externals
-manage_externals.log
+# directories checked out by git-fleximod, and other files created
+# by git-fleximod
+git-fleximod.log
```

Comment:
Here and elsewhere:
I did a manual "find and replace" so please let me know if some of the documentation needs updating to be correct.